db/seeds.rb

```diff
@@ -5,3 +5,16 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+ActiveRecord::Base.transaction do
```

Comment:
> Should this only have a guard to work in development environments?

Well, to tell the truth I would rather not, because db:seed should never run in production, but I am open to have the guard if you want to. Let me know :D

Reply:
Some apps need production seeds – there's a valid use-case for this. But it's not a big deal. The worst case scenario is that some dummy data enters production.